web/src/styles/ButtonOutline.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import { ButtonOutline, util } from 'rebass';
```

Comment:
Where did you find docs about this `util` thing?  The rebass docs seem to be missing a lot of stuff that isn't the components, and even with the components, it's basically just "here they are."  😕

Reply:
found them just by reading the source code... yes, the docs are certainly lacking and i'm also worried that this library is getting a little stale too... i think in general `rebass` has been helpful during this prototyping phase, and i'm not sure i __love__ it / will even __like__ it long-term. we should talk more about this next sprint

Reply:
Ah good, glad to hear that (sorta...  you know what I mean).  I'm not exactly loving it either.  It seems _okay_, but I feel pretty confident that there are better ways forward.

Reply:
agreed 👍

Reply:
i'm happy we tried it, but i think i'm gonna be happy to change it up too